src/Microsoft.NET.Test.Sdk.targets

```diff
@@ -24,16 +24,25 @@
     <GenerateProgramFile Condition="'$(GenerateProgramFile)' == ''">true</GenerateProgramFile>
   </PropertyGroup>
 
-  <!-- Output type for dotnet core test project should be exe. For full CLR it is Library by default.
-       This should be added in props file.
-       Issues: https://github.com/dotnet/roslyn-project-system/issues/268,
-       https://devdiv.visualstudio.com/DevDiv/_workitems?id=375688&_a=edit
+  <!--
+     Output type for dotnet core test project should be exe. For full CLR it is Library by default.
+     This should be added in props file.
+     Issues: https://github.com/dotnet/roslyn-project-system/issues/268,
+     https://devdiv.visualstudio.com/DevDiv/_workitems?id=375688&_a=edit
   -->
   <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
     <OutputType>Exe</OutputType>
   </PropertyGroup>
 
   <!--
+     In IDE scenario for full CLR projects default test platform is v1, which expects full pdbs for source information.
+     This can be removed once TPv2 is default for full CLR. Related issue https://github.com/Microsoft/vstest/issues/373.
+   -->
+  <PropertyGroup>
+    <DebugType Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">Full</DebugType>
```

Comment:
It is not a good idea, as @smadala mentioned in comments we should remove it once we've TPv2 OM on test explorer as well :)